pkg/ui/console/summary.go

```diff
@@ -0,0 +1,85 @@
+package console
+
+import (
+	"strconv"
+
+	"github.com/cloudquery/cloudquery/pkg/core"
+	"github.com/cloudquery/cloudquery/pkg/ui"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/spf13/viper"
+)
+
+func PrintProviderSummary(summary *core.ProviderFetchSummary) {
+	s := emojiStatus[ui.StatusOK]
+	if summary.Status == core.FetchCanceled {
+		s = emojiStatus[ui.StatusError] + " (canceled)"
+	}
+	key := summary.Name
+	if summary.Name != summary.Alias {
+		key = summary.Name + `(` + summary.Alias + `)`
+	}
+	diags := summary.Diagnostics().Squash()
+	const (
+		fetchSummary = "Provider %s fetch summary: %s Total Resources fetched: %d"
+	)
+	ui.ColorizedOutput(
+		ui.ColorHeader,
+		fetchSummary,
```

Comment:
```suggestion
	ui.ColorizedOutput(
		ui.ColorHeader,
		"Provider %s fetch summary: %s Total Resources fetched: %d",
```

[non-blocking] Is this `const` used someplace else?

Reply:
no, just this block of code